README.md

```diff
@@ -22,9 +22,15 @@ directly without another `pip install`.
 
 ### CommCare HQ OAuth Integration
 
-- Create an OAuth application on CommCare HQ using Django Admin
-  `<hq_host>/admin/oauth2_provider/application/`. Use
-  `<superset_host>/oauth-authorized/commcare` as the redirect URL.
+- Create an OAuth application on CommCare HQ using Django Admin at the URL
+  `<hq_host>/admin/oauth2_provider/application/` with the following settings:
+  - Redirect URIs: `<superset_host>/oauth-authorized/commcare`
+  - Leave "Post logout redirect URIs" empty.
+  - Client type: Confidential
+  - Authorization grant type: Client credentials
```

Comment:
I used "Authorization code" for this.

Reply:
:+1: That is the correct grant type. 0eb8f528e57850044e9ffcc3e8bb203040771cfb